JavaScript/packages/recognizers-date-time/src/resources/spanishDateTime.ts

```diff
@@ -250,10 +251,10 @@ export namespace SpanishDateTime {
     export const MinusOneDayTerms = [ "ayer","ultimo dia" ];
     export const PlusTwoDayTerms = [ "pasado mañana","dia despues de mañana" ];
     export const MinusTwoDayTerms = [ "anteayer","dia antes de ayer" ];
-    export const MonthTerms = [ "mes","meses" ];
+    export const MonthTerms = [ "mes","meses","mes pasado","mes pasados" ];
```

Comment:
Such relative terms should not be added to the Terms lists. For example, in English, the lists are clean.